README.md

```diff
@@ -101,11 +116,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
```

Comment:
The first 3 you checked are correct, but this line doesn't work because `document.getElementsByClassName` returns an array, and you can't call `innerHTML` on arrays, only individual elements inside of it.